config.json

```diff
@@ -920,6 +936,142 @@
         "nugetId": "Xamarin.Firebase.ProtoliteWellKnownTypes",
         "dependencyOnly": false
       },
+      {
+        "groupId": "com.google.android.play",
+        "artifactId": "core",
+        "version": "1.9.0",
+        "nugetVersion": "1.9.0",
+        "nugetId": "Xamarin.Google.Android.Play.Core",
+        "dependencyOnly": false
+      },
+      {
+        "groupId": "com.google.mlkit",
+        "artifactId": "common",
+        "version": "16.0.0",
+        "nugetVersion": "16.0.0",
+        "nugetId": "Xamarin.GoogleFirebase.ML.Kit.Common",
```

Comment:
Have these packages already shipped? Or can we still change this name?

The iOS packages call it [Xamarin.Firebase.iOS.MLKit](https://www.nuget.org/packages/Xamarin.Firebase.iOS.MLKit/), which also seems more in line with our other Firebase packages, like `Xamarin.Firebase.Storage`.

Maybe we could name this `Xamarin.Firebase.Android.MLKit.*`?

We also have `Xamarin.GooglePlayServices.MLKit.BarcodeScanning`, so we should definitely try to be consistent with `MLKit` vs. `ML.Kit`.

Same goes for the namespaces which it would be nice if they match our existing pattern.

Reply:
![image](https://user-images.githubusercontent.com/179295/106364450-b39ec080-62f4-11eb-8d2e-7d7c5b8840e2.png)

Or if these are the new non-Firebase versions, maybe we remove "Firebase" from the names to prevent confusion.

Reply:
There was so much changes. I am OK with everything/anything.
At the begining Firebase denoted cloud versions IIRC. Later it changed to Google MLKit. 

I am not sure when iOS bindings were done, but I am OK with anything

Reply:
Nothing was published (yet)

Reply:
While you were reviewing I already changed ML.Kit to MLKit